sdk/core/core/inc/az_span.h

```diff
@@ -282,6 +282,7 @@ AZ_NODISCARD az_result az_span_to_uint32(az_span span, uint32_t* out_number);
  */
 AZ_NODISCARD int32_t az_span_find(az_span source, az_span target);
 
+AZ_NODISCARD int32_t az_span_find2(az_span source, int32_t start, az_span target);
```

Comment:
is this supposed to be here?

Reply:
Oh no! please disregard this!

Reply:
Just pushed an update, those changes in SDK/core are gone now. Sorry for that.